libstarlark/src/main/java/net/starlark/java/eval/StarlarkList.java

```diff
@@ -455,6 +455,17 @@ public void append(Object item) throws EvalException {
     addElement((E) item); // unchecked
   }
 
+  @StarlarkMethod(name = "reverse", doc = "Reverses the order of the list items.")
```

Comment:
We shouldn't modify the StarlarkList file. You can use reversed(list) instead.

Reply:
I thought part of our mission with Larky is to make it work as close to python as possible. Is there a reason we shouldn't have this built in functionality? 

I can fix the code, I'm just wondering why we're not gonna implement something built-in to python.

Reply:
StarlarkList is owned upstream by Google's bazel team. We don't want to maintain the differences. 

The only reason it diverges slightly is because we added bytes support and that was not added in upstream bazel, but we should absolutely merge it upstream.

StarlarkList modifications should go to bazelbuild/starlark first and if approved, we can add it. Otherwise, we shouldn't add it.

Also, I don't see the difference between reversed(list) vs list.reverse. Both are valid python.

Reply:
Ah, that makes perfect sense. Thank you for the explanation.